app/src/main/res/values/strings.xml

```diff
@@ -519,7 +519,7 @@
   <string name="bloodsugarentry_error_higher_limit">Value cannot be more than 1000 mg/dl</string>
   <string name="bloodsugarentry_error_higher_limit_hba1c">Value cannot be more than 25%</string>
   <string name="bloodsugarentry_error_empty">Value cannot be empty</string>
-  <string name="bloodsugarentry_error_date_invalid_pattern">Enter a valid date in DD/MM/YY</string>
+  <string name="bloodsugarentry_error_date_invalid_pattern">Enter a valid date in DD/MM/YYYY</string>
```

Comment:
The pivotal story says that :
"Backdating a BP error message says the date must be DD/MM/YYYY, but it actually requires a 2 digit date".
It is a 2 digit date. Why are we changing it to 4 digits?

Reply:
The pivotal story says that right now the app has only 2 digits but it must actually be YYYY.

Reply:
No. We enter 2 digits. Can you please check the app?

Reply:
You are right. Confirmed with Pragati that this feature has to be changed to accept 4 digits. So I'll skip this commit and make this change in another PR.